readthedocs/projects/models.py

```diff
@@ -225,6 +230,35 @@ class Project(models.Model):
         ),
     )
 
+    urlpattern = models.CharField(
+        _("Custom URL pattern for multi version and single version projects"),
```

Comment:
Seems odd to include multi & single version in the docstring. Isn't that all projects?